### PR TITLE
add new site

### PR DESCRIPTION
**Pull request by nibunasah**
add new site

### CODE REVIEW

**Review by Airborn22 — APPROVED** [review]
LGTM

**Comment by Arumankumang** [comment]
I'm still studying, please help

File: related_website_sets.JSON [hunk 1]
```diff
@@ -806,7 +806,19 @@
...
       "https://thirdspace.org.au": "Similar content served in slightly different formats to different audiences"
     }
-  },{
+  },
+  {
+    "contact": "admin@smpn106jkt.sch.id",
+    "primary": "https://p106.net",
+    "associatedSites": [
+      "https://smpn106jkt.sch.id"
+    ],
```
Comment (Parasos) [inline related_website_sets.JSON:815]: related_website_sets.JSON

**Review by Parasos — COMMENTED** [review]
related_website_sets.JSON

File: related_website_sets.JSON [hunk 2]
```diff
@@ -806,7 +806,19 @@
     "rationaleBySite": {
       "https://thirdspace.org.au": "Similar content served in slightly different formats to different audiences"
     }
-  },{
+  },
+  {
+    "contact": "admin@smpn106jkt.sch.id",
```
Comment (Parasos) [inline related_website_sets.JSON:811]: https://support.google.com/accounts/answer/7675428?hl=ru

File: related_website_sets.JSON [hunk 3]
```diff
@@ -823,7 +835,7 @@
         "https://indiatoday.in": "IndiaToday.in brings you news, analysis, explainers and opinion on a variety of subjects including politics, crime, civic issues, business, sports, technology, science, heatlh, lifestyle, and much more",        
         "https://gnttv.com": " Good News Today is a website that focuses on the bright side of life and showcases stories of triumphs and positivity.",
         "https://indiatodayne.in": "India Today NE is dedicated to covering news and views of India’s north-eastern region which includes eight states—Assam, Arunachal Pradesh, Nagaland, Manipur, Mizoram, Tripura, Meghalaya and Sikkim"
-}
-}
+    }
```
Comment (Parasos) [inline related_website_sets.JSON:838]: related_website_sets.JSON

**Review by Parasos — COMMENTED** [review]
Submission_Guidelines.mdhttps://accounts.google.com/v3/signin/rejected?continue=https://chrome.google.com/sync&ddm=0&dsh=S1191280729:1720127362912122&flowEntry=ServiceLogin&flowName=GlifWebSignIn&followup=https://chrome.google.com/sync&ifkv=AS5LTAQLy8WVMkj4746iRWpK2EpU2J76SrSyxREUtebrkY6wLHrHcu9QOy6HAyl1EO8WqITixPMZ5g

**Pull request merged**